plenum/test/restart/test_restart_node_with_view_changes.py

```diff
@@ -81,31 +81,27 @@ def patch_on_view_change_started(node, msg, frm):
     lagging_node.nodeMsgRouter.add((ViewChangeStartMessage, processor))
 
     # Delay CurrentState messages on lagging_node to delay propagate primary
-    with delay_rules(lagging_node.nodeIbStasher, cs_delay()):
+    with delay_rules(lagging_node.nodeIbStasher, msg_rep_delay()):
         # Add lagging_node to pool
         looper.add(lagging_node)
         txnPoolNodeSet[-1] = lagging_node
         looper.run(checkNodesConnected(txnPoolNodeSet))
         looper.run(
             eventually(lambda: assertExp(len(lagging_node.nodeIbStasher.delayeds) >= 3)))
 
-        # Change viewNo in received CurrentStates.
-        for cs in lagging_node.nodeIbStasher.delayeds:
-            cs[0][0].viewNo = 1
-            for vcd in cs[0][0].primary:
-                vcd[f.VIEW_NO.nm] = 1
-
         # Start ViewChange (0 -> 1)
         for n in rest_nodes:
             n.view_changer.on_master_degradation()
 
-        # Wait View Change start with InstanceChange messages on lagging_node
+        # Lagging node still did not catchup, so it can't participate and process I_CH
         looper.run(
             eventually(
-                lambda: assertExp(len(view_change_started_messages) == 1)))
+                lambda: assertExp(len(view_change_started_messages) == 0)))
+
+    # Lagging node catches up till old view
     looper.run(
         eventually(
-            lambda: assertExp(lagging_node.viewNo == start_view_no + 1)))
+            lambda: assertExp(lagging_node.viewNo == start_view_no)))
```

Comment:
It looks like the test intention has changed. Previously it was assumed that it starts and finishes a view change, but now it's not .

Reply:
The logic of initial view setting, that is tested there, is changed much, so probably this test have no much value for now..